src/scss/mixins/_single-post-content.scss

```diff
@@ -114,7 +96,7 @@
     }
 
     p {
-      font-family: $font-family-sans-serif;
+      font-family: $font-family-slab-serif;
       font-size: $blockquote-font-size;
```

Comment:
Since we're changing the font family for these blockquotes, it may make sense to bump up the font size a bit, since Athena's slab serif font ends up feeling a little smaller than the sans-serif font.

There are some variables included in Athena that help balance font sizes of lead text, blockquotes, etc between all the available font families.  Maybe try changing `$blockquote-font-size` and `$blockquote-font-size-md` to `$font-slab-serif-blockquote-size` and `$font-slab-serif-blockquote-size-md`, respectively?

https://github.com/UCF/Athena-Framework/blob/master/src/scss/_variables.scss#L557-L558